setup.py

```diff
@@ -0,0 +1,28 @@
+# encoding: utf8
+from setuptools import setup
+
+setup(
+    name='python-gp',
+    version='0.1',
+    author='Jakob Jordan, Maximilian Schmidt',
+    author_email='j.jordan@fz-juelich.de',
+    description=('Cartesian genetic programming in Python.'),
+    license='GPLv3',
+    keywords='genetic programming',
+    url='https://github.com/jakobj/python-gp',
+    python_requires='>=3.6, <4',
+    install_requires=['sympy', 'torch', 'numpy', ],
```

Comment:
shouldn't we read this from requirements.txt?

Reply:
requirements file is introduced in #23 , so let's merge #23 first and I'll modify here then.